pkg/processor/trigger/kafka/trigger.go

```diff
@@ -132,6 +132,13 @@ func (k *kafka) Start(checkpoint functionconfig.Checkpoint) error {
 			k.ctx = context.Background()
 			k.Logger.DebugWith("Starting to consume from broker", "topics", k.configuration.Topics)
 
+			// signal workers to continue event processing
+			if err = k.SignalWorkersToContinue(); err != nil {
+				k.Logger.WarnWith("Failed to signal worker to continue event processing",
+					"err", err)
+				continue
```

Comment:
if it is not being able to signaling after 2-3 times I would suggest to panic and let the entire function to restart, perhaps a zombie process is there and we dont want to get stuck with kafka stream not being consumed